.storybook/preview.tsx

```diff
@@ -1,6 +1,9 @@
+import React from 'react';
 import { addDecorator } from '@storybook/react';
 import { MemoryRouter } from 'react-router-dom';
 
+import '../src/app/globals.css';
```

Comment:
Not related to the upgrade, but I noticed that I forgot to load global stylesheets in storybook.